tests/performance/pka_test_performance.c

```diff
@@ -114,20 +114,22 @@ static pka_barrier_t thread_end_barrier;
 // *FIXME*
```

Comment:
I think we should get rid of this define i.e., "ARMV8_PMU_ENBALE". It is useless, right?

Reply:
Perhaps, we should get rid of the PM counters and keep the system counters, instead. This way we remove all extra dependencies